jumpscale/servers/threebot/threebot.py

```diff
@@ -225,6 +228,14 @@ def module(self):
     def base_url(self):
         return j.sals.fs.join_paths("/", self.name)
 
+    @property
+    def config(self):
+        if not self._config:
+            self._config = self.load_config()
+            if self._config:
+                self.name = self._config["name"]
```

Comment:
why you reset the name here ?
it should be like 
```python
if self._config is None:
    self._config = self.load_config()
return self._config
```

Reply:
because before name is in init and name is get from config , so when config is exist and loaded get name otherwise not change it .

Reply:
we already now get the name using this line
```j.sals.fs.basename(path.rstrip('/'))```